packages/kotti-ui/source/kotti-field-file-upload/utils/common.ts

```diff
@@ -0,0 +1,57 @@
+import dayjs from 'dayjs'
+import filesize from 'filesize'
+
+import { ISO8601_SECONDS } from '../constants'
+import { KottiFieldFileUpload } from '../types'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const nanoid = require('nanoid')
+
+export const formatFileSize = (fileSize: number): string =>
+	filesize(fileSize, { round: 1, spacer: '' })
+
+export const formatFileTypes = (
+	validFileTypes: Array<string> | null,
+): string | null => validFileTypes?.join(', ').toUpperCase() ?? null
+
+export const getFileInputAccept = (
+	validFileTypes: Array<string> | null,
+): string | undefined =>
+	validFileTypes?.length ? `.${validFileTypes.join(',.')}` : undefined
+
+export const getFileInfo = (
+	file: KottiFieldFileUpload.FileItem,
+): KottiFieldFileUpload.FileInfo => ({
+	id: file.id,
+	name: file.name,
+	progress: 0,
+	size: file.blob.size,
+	status: KottiFieldFileUpload.Status.NOT_STARTED,
+})
+
+export const getFileItem = (file: File | Blob): KottiFieldFileUpload.FileItem =>
+	file instanceof File
+		? {
+				blob: file,
+				id: nanoid(),
+				name: file.name,
+		  }
+		: {
+				blob: file as File,
+				id: nanoid(),
+				name: `${dayjs().format(ISO8601_SECONDS)}.png`,
+		  }
+
+export const getValidFileTypes = (
```

Comment:
I don't understand what this is supposed to do, so either refactor it or add a JSDoc